cns/restserver/restserver_test.go

```diff
@@ -618,6 +618,29 @@ func TestCreateNetworkContainer(t *testing.T) {
 
 }
 
```

Comment:
Instead of creating a separate test, can you move this under the test we have to create container? I think it was creating webApps; I added the JobObject type subtest in the same test